src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -781,6 +738,18 @@ static int process_loader_partitions(DissectedPartition *esp, DissectedPartition
         assert(esp);
         assert(xbootldr);
 
+        /* If any paths in fstab look similar to our favorite paths for ESP or XBOOTLDR, we just exit
+         * early. We also don't bother with cases where one is configured explicitly and the other shall be
+         * mounted automatically. */
+
+        r = fstab_has_mount_point_prefix_strv(STRV_MAKE("/boot", "/efi"));
+        if (r > 0) {
+                log_debug("Found path in fstab starting with either /boot or /efi, skipping ESP and XBOOTLDR mounts.");
```

Comment:
we generally suffix paths that refer to dirs with a dash in docs and messages. THus please change these paths here to /boot/ and /efi/.

Also, it's not about finding paths, it's about finding *mounts* there.

Maybe: "Found mounts in fstab below the /boot/ or /efi/ hierarchies, skipping ESP and XBOOTLDR mounts."

Reply:
I'll go with the second suggestion.